lib/EncodeImagesStage.js

```diff
@@ -4,20 +4,32 @@ var Promise = require('bluebird');
 
 Jimp.prototype.getBufferAsync = Promise.promisify(Jimp.prototype.getBuffer);
 
-module.exports = encodeImages;
+module.exports = EncodeImagesStage;
+
+/**
+ * Encodes Jimp images in pipeline extras to buffers.
+ *
+ * The glTF asset must be initialized for the pipeline.
+ */
+function EncodeImagesStage() {
+    this.name = 'encodeImages';
+    this.before = ['quantizeAttributes'];
+    this.after = [];
```

Comment:
My understanding from the issue comments was that the `before` and `after` would be used to run any stages that are required before/after by the current stage.

For example if stage `Y` requires `X` before and `Z` after, then before would be `X` and after would be `Y`.
If we have stages `A`, `B`, `C` that can all run independently, then none would be before or after in this case.

-----
I was planning on doing the dependency graph last - once all the stages have been made asynchronous. What do you think of this plan?